tests/integration-tests/tests/storage/snapshots_factory.py

```diff
@@ -184,7 +179,7 @@ def _launch_instance(self, ami_id, subnet):
                     "SubnetId": subnet.id,
                     "DeviceIndex": 0,
                     "AssociatePublicIpAddress": True,
-                    "Groups": [self._get_security_group_id()],
+                    "Groups": [self.security_group_id],
```

Comment:
Should this be calling the function, as it was before?

Reply:
I don't think so, `_launch_instance` is called after `self.security_group_id = self._get_security_group_id()`, and all the `_get_security_group_id()` function does before in context of `_launch_instance` is return the id of SG that was created.

Ideally we should refactor the snapshot factory slightly to move most of attribute initialization in `__init__`, but we will probably do this later